output.c

```diff
@@ -318,7 +324,10 @@ Output_t* Output_Create(bool IsNULL,
 	u32 CPUCnt 	= 0;
 	for (int i=0; i < 32; i++)
 	{
-		pthread_create(&O->PushThread[i], NULL, Output_Worker, (void*)O); 
+		ThreadData_t *T = malloc(sizeof(ThreadData_t));
```

Comment:
pls create a list of thread data in the Output_t structure. That way theres no dangling pointers left anywhere where a toplevel process might want to walk all structures.

Reply:
e.g OutputThread_t* Out->Thread[128];

My code is not great here... it evolved from a single threaded version, alot of the fields in Output_t should go into OutputThread_t e.g. everything that uses CPUID as an index. Once this is pushed, I will clean it up and move things to the OutputThread_t structure.

Reply:
Added new OutputThread_t and took array of it withing Output structure.
Once we close with the persistent connection topic, we may slowly move all the thread related variables to that structure.